app/src/components/forms/roles/AddRoleCategoryForm.vue

```diff
@@ -7,15 +7,29 @@
   <div class="max-h-[70vh] overflow-auto">
```

Comment:
```suggestion
  <div class="max-h-[70vh]">
```
![image](https://github.com/user-attachments/assets/258423c1-1d3d-450c-ba9c-04d60512f1c8)

Reply:
Okay, thanks